### ISSUE TITLE
Should this check for missing definitions?

### ISSUE THREAD

**Issue by jacobweber** [swaggest/php-swagger2-schema#8]
Hi. I was trying this out to validate some Swagger 2.0 specs that I auto-generate. I don't see any issues here, so apologies if this library isn't meant for public consumption.

I downloaded the [petstore](https://github.com/OAI/OpenAPI-Specification/blob/main/examples/v2.0/json/petstore-simple.json) example from OpenAPI as a test.

It passes validation:
```php
<?php
require __DIR__ . '/vendor/autoload.php';
$json = json_decode(file_get_contents(__DIR__ . '/petstore-simple.json'));
$schema = Swaggest\SwaggerSchema\SwaggerSchema::import($json);
```

But if I change the petstore JSON to reference an invalid definition, it still passes.

For example, I changed line 176 to:
```diff
   "definitions": {
     "Pet": {
       "type": "object",
       "allOf": [
         {
-           "$ref": "#/definitions/NewPet"
+           "$ref": "#/definitions/foo"
        },
```

[swagger-parser](https://apitools.dev/swagger-parser/online/) displays an error when I do this, and it seems to use the same schema. But maybe it does some extra validation. Should this library catch this?

Or if I insert this before line 183, it also passes:
```diff
         {
           "required": [
             "id"
           ],
           "properties": {
+            "bad": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Bad"
+              }
+            },
             "id": {
               "type": "integer",
               "format": "int64"
             }
           }
         }
```

I was originally trying to use php-json-schema to validate my specs against the [Swagger 2.0 schema](https://github.com/OAI/OpenAPI-Specification/tree/main/schemas/v2.0). The older version 0.12.7 caught the second error but not the first, but the current version doesn't catch either. Which is why I tried this library.

**Comment by vearutop** [comment]
Thank you for raising this issue, please try `v0.2.5`.

**Comment by jacobweber** [comment]
That was fast! Thanks so much for looking into it. This seems to fix the issue for the first case, but not the second. It passes validation when you have an array property with a bad $ref for its "items" definition. Or if the "items" is a good ref, it still passes validation when you have bad refs inside the definition that it points to. Here's an example with both:

```json
{
    "swagger": "2.0",
    "info": {
        "title": "test",
        "version": "1.0.0"
    },
    "paths": {
        "/test": {
            "get": {
                "summary": "test",
                "responses": {
                    "200": {
                        "description": "successful response",
                        "schema": {
                            "$ref": "#/definitions/response"
                        }
                    }
                }
            }
        }
    },
    "definitions": {
        "response": {
            "properties": {
                "foo": {
                    "type": "array",
                    "items": {
                        "$ref": "#/definitions/good"
                    }
                },
                "bar": {
                    "type": "array",
                    "items": {
                        "$ref": "#/definitions/missing1"
                    }
                }
            }
        },
        "good": {
            "properties": {
                "foo": {
                    "$ref": "#/definitions/missing2"
                }
            }
        }
    }
}
```

**Comment by vearutop** [comment]
Hopefully the issue is fixed with `v0.2.6`. :)

**Comment by jacobweber** [comment]
Yep, validation failed! Thanks again.
